Artem Sushko/Lesson7/Lesson7.Homework/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿internal class Program
+{
+    static bool Compare(string str1, string str2)
+    {
+        if (str1 == str2)
+            return true;
+
+        return false;
+    }
+
+    static string Analyze(string str3)
+    {
+        var IsSymbol = 0;
+        var IsLetter = 0;
+        var IsDigit = 0;
+        for (int i = 0; i < str3.Length; i++)
+        {
+            if (char.IsLetter(str3[i]))
+            {
+                IsLetter++;
+            }
+            else if (char.IsDigit(str3[i]))
+            {
+                IsDigit++;
+            }
+            else IsSymbol++;
+        }
+        string res = $"Amount of Letters is {IsLetter}\nAmount of Digits is {IsDigit}\nAmount of another special characters is {IsSymbol}";
+        return res;
+    }
+
+    static string Sort(string str4)
+    {
+        char[] strchar = str4.ToCharArray();
+        char tmp;
+        for (int i = 0; i < strchar.Length - 1; i++)
+        {
+            for (int j = 0; j < strchar.Length - 1 - i; j++)
+            {
+                if (strchar[j] > strchar[j + 1])
+                {
+                    tmp = strchar[j + 1];
+                    strchar[j + 1] = strchar[j];
+                    strchar[j] = tmp;
+                }
+            }
+        }
+        String str5 = new string(strchar);
+        return str5;
+    }
+
+    static string Duplicate(string str4)
+    {
+        string res = "";
```

Comment:
better to use array or at least StringBuilder to not create a lot of strings during metod invocation